maintainers/maintainer-list.nix

```diff
@@ -3105,6 +3105,12 @@
     githubId = 17111639;
     name = "Devin Singh";
   };
+  devusb = {
+    email = "mhelton@devusb.us";
+    github = "devusb";
+    githubId = 4951663;
+    name = "Morgan Helton";
+  };
```

Comment:
Please put this in a [separate commit](https://nixos.org/manual/nixpkgs/stable/#var-meta-maintainers) `maintainers: add devusb` so there are exactly 2 commits in this PR.

Reply:
Done.